CHANGELOG.md

```diff
@@ -0,0 +1,210 @@
+# Changelog
+
+## Release 0.39
+
+* Fix XPath computation  
+* Fix Check Path field with latest Jenkins version in configuration  
+* Upgrade to latest Jenkins LTS (1.554.1)
+
+## Release 0.38
+
+* Fix
+[JENKINS-17591](https://issues.jenkins-ci.org/browse/JENKINS-17591) -
+FSTrigger fails to poll when build node labels change
+
+## Release 0.37
+
+* Fix
+[JENKINS-18658](https://issues.jenkins-ci.org/browse/JENKINS-18658) -
+NPE in FSTrigger upon configuration save
+
+## Release 0.36
+
+* Fix
+[JENKINS-17641](https://issues.jenkins-ci.org/browse/JENKINS-17641) -
+Unknown field 'logEnabled' in org.jenkinsci.lib.xtrigger.XTriggerCause
+
+## Release 0.35
+
+* Fix
+[JENKINS-16809](https://issues.jenkins-ci.org/browse/JENKINS-16809) -
+Plugins FSTrigger/Envinject, NPE while loading jobs  
+  * Update to xtrigger-lib 0.20  
+  * Update to envinject-lib 1.17
+
+## Release 0.34
+
+* Fix NullPointerException on polling action  
+* Upgrade to envinject-lib 1.11  
+* Upgrade to xtrigger-lib 0.18
+
+## Release 0.33
+
+* Fix potential NullPointer exception at startup (envinject-lib 1.8/
+xtrigger-lib 1.5)
+
+## Release 0.32
+
+* Upgrade to xtrigger-lib 0.14 (more logs)
+
+## Release 0.31
+
+* Fix
+[JENKINS-12176](https://issues.jenkins-ci.org/browse/JENKINS-12176) -
+Unable to delete a job that has a fstrigger  
+* Upgrade to xtrigger-lib 0.13
+
+## Release 0.30
+
+* Fix reponed
+[JENKINS-12924](https://issues.jenkins-ci.org/browse/JENKINS-12924) -
+FSTrigger triggers builds on jenkins restart
+
+## Release 0.29
+
+* Fix
+[JENKINS-12924](https://issues.jenkins-ci.org/browse/JENKINS-12924) -
+FSTrigger triggers builds on jenkins restart
+
+## Release 0.28
+
+* Update to xtrigger-lib 0.8 (fix
+[JENKINS-12888](https://issues.jenkins-ci.org/browse/JENKINS-12888))
+
+## Release 0.27
+
+* Fix
+[JENKINS-12865](https://issues.jenkins-ci.org/browse/JENKINS-12865) -
+[ERROR](https://wiki.jenkins.io/display/JENKINS/FSTrigger+Plugin#) -
```

Comment:
You can replace https://wiki.jenkins.io/display/JENKINS/FSTrigger+Plugin# with https://plugins.jenkins.io/fstrigger/
as it is already redirecting to later url